CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 15.6.2
+
+- Fixed: `alpha-value-notation` false negatives for `oklab()`, `oklch()`, and `color()` ([#6844](https://github.com/stylelint/stylelint/pull/6844)) ([@romainmenke](https://github.com/romainmenke)).
+- Fixed: `declaration-block-no-redundant-longhand-properties` `autofix` with `cubic-bezier()` ([#6841](https://github.com/stylelint/stylelint/pull/6841)) ([@romainmenke](https://github.com/romainmenke)).
```

Comment:
We should fix the following before merging it:

```suggestion
- Fixed: `declaration-block-no-redundant-longhand-properties` autofix with `cubic-bezier()` ([#6841](https://github.com/stylelint/stylelint/pull/6841)) ([@romainmenke](https://github.com/romainmenke)).
```

Reply:
Ah! Sorry for not catching that on review!